dataconnect/README.md

```diff
@@ -0,0 +1,41 @@
+Firebase DataConnect Quickstart
+=======================================
+
+Introduction
+------------
+
+This is a sample app for the preview version of the Firebase DataConnect.
+This will not work if you don't have access to the preview.
+
```

Comment:
Sign up for early access [here](https://firebase.google.com/products/data-connect)